components/switchCampuses/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import { Button } from 'react-native-elements';
+import styles from './styles';
+
+export default class SwitchCampuses extends Component {
```

Comment:
Can this component appear when a user decides to enter a search ?  then when the user starts typing, it would hide this component with the search results

Reply:
After contemplation, I came with the conclusion that this should stay at this level for now as other components will use this. In the future, let's create a wrapper for the search bar so that it handles all the navigation requests. Right now, this wrapper is the "Home" component